crud/common/call.lua

```diff
@@ -32,22 +31,19 @@ local function call_impl(vshard_call, func_name, func_args, opts)
         end
     end
 
-    local futures_by_replicasets = {}
-    local call_opts = {is_async = true}
+    local futures = {}
     for _, replicaset in pairs(replicasets) do
-        local future = replicaset[vshard_call](replicaset, func_name, func_args, call_opts)
-        futures_by_replicasets[replicaset.uuid] = future
+        local future = replicaset[vshard_call](replicaset, func_name, func_args, {
+            timeout = timeout,
+            is_async = true,
```

Comment:
`timeout` will be ignored if `is_async=true` is specified. Correct way to handle such case were described below - https://github.com/tarantool/crud/pull/38/files#diff-d0dbf8e86bcf0f98f008ed20030bfdc2c3f106a245d3343f7cf3acac36b6f921L43-L48

See also https://github.com/tarantool/doc/issues/1710